x-pack/plugins/security_solution/server/endpoint/routes/resolver/utils/pagination.ts

```diff
@@ -31,7 +31,7 @@ export type SortFields = [
   {
     '@timestamp': string;
   },
-  { [x: string]: string }
+  { [x: string]: { order: string; unmapped_type?: string } }
```

Comment:
Had to look this up, but it makes sense based on the docs: https://www.elastic.co/guide/en/elasticsearch/reference/current/sort-search-results.html#_ignoring_unmapped_fields

Reply:
I had basically every windows related integration installed when trying to collect sysmon data in my local setup, and after it eventually started flowing, I noticed calls to /api/endpoint/resolver/events would sometimes fail due to the event.id not being mapped in a few esoteric windows system indices, adding this made the error not happen. Don't think it'll ever have any unintended consequences, as the indices events come from should have that field mapped already, and even if not, this is just for a tiebreak if timestamps are equal.